CHANGELOG.adoc

```diff
@@ -21,6 +21,11 @@ https://github.com/oxidecomputer/dropshot/compare/v0.10.1\...HEAD[Full list of c
 
 === Other notable changes
 
+* The `RequestContext` type now contains the `operation_id`, the name of the
```

Comment:
Is this a breaking change because people could be matching the contents of RequestContext?

we often put the PR number in here